apps/condo/domains/banking/components/BankAccountInfo.tsx

```diff
@@ -45,6 +46,19 @@ export const BankingInfo: React.FC<IBankAccountInfo> = ({ bankAccount }) => {
         return { width: breakpoints.MOBILE_SMALL && !breakpoints.DESKTOP_LARGE ? '100%' : '425px' }
     }, [breakpoints])
 
+    // Note(YEgorLu): could use just bankAccount.name after migrating organization names to empty bankAccount names
+    const bankAccountName = useMemo(() => {
+        const bankAccountName: string | undefined = get(bankAccount, 'name')
```

Comment:
you can add a default empty string and do not declare type as string or undefined
get(bankAccount, 'name', '')